core/server/models/api-key.js

```diff
@@ -26,6 +28,50 @@ const createSecret = (type) => {
     return crypto.randomBytes(bytes).toString('hex');
 };
 
+const addAction = (model, event, options) => {
```

Comment:
@kevinansfield This is pretty dirty and basically just copied over from base model [here](https://github.com/TryGhost/Ghost/blob/master/core/server/models/base/index.js#L103) as we didn't seem to expose `addAction` for non db events actions yet. 😬 Still thinking of putting this a better way, though let me know in case i missed a pattern we are already using to add new events like this

Reply:
Seems fine for now. Something to come back to next time we have to create actions outside of events though